### PR TITLE
fix(cli): add `--datasource-provider` to `init` command

### PR DESCRIPTION
**Pull request by Jolg42**
Related prisma/prisma#7667

Todo:
- add param in help 
- add param in docs

For release notes

Original Issue 
https://github.com/prisma/prisma/issues/2779

### `prisma init` now accepts a `--datasource-provider` argument

The [`prisma init`](https://www.prisma.io/docs/reference/api-reference/command-reference#init) command now accepts a `--datasource-provider` argument that lets you configure the default `provider` for the initially generated `datasource` block in your Prisma schema. The possible values for this argument are equivalent to the [allowed values](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#fields) for the `provider` field on `datasource` blocks:

- `postgresql`
- `mysql`
- `sqlite`
- `sqlserver` ([Preview](https://github.com/prisma/prisma/issues/4039))

Here's an example that shows how to configure the initial Prisma schema skeleton with a SQLite database:

```
npx prisma init --datasource-provider sqlite
```

### CODE REVIEW

**Review by millsp — APPROVED** [review]

File: src/packages/cli/src/Init.ts [hunk 1]
```diff
@@ -102,7 +101,7 @@ export class Init implements Command {
       '--help': Boolean,
       '-h': '--help',
       '--url': String,
-      // '--provider': String,
+      '--provider': String,
```
Comment (janpio) [inline src/packages/cli/src/Init.ts:104]:
```suggestion
      '--datasource-provider': String,
```

File: src/packages/cli/src/Init.ts [hunk 2]
```diff
@@ -87,13 +87,29 @@ export class Init implements Command {
     return new Init()
   }
 
-  // TODO add --provider & --url
   private static help = format(`
-    Setup a new Prisma project
+  Setup a new Prisma project
```
Comment (janpio) [inline src/packages/cli/src/Init.ts:91]: Changing indentation will not have an effect on the CLI output?

**Review by janpio — APPROVED** [review]
<3

**Review by millsp — APPROVED** [review]

File: src/packages/cli/src/Init.ts (hunk 2 above)
Comment (Jolg42) [reply src/packages/cli/src/Init.ts:91]: No, and here is the new output 
Screen Shot 2021-06-25 at 19 35 16

Comment (Jolg42) [reply src/packages/cli/src/Init.ts:91]: I spotted a misalignment, I'll fix

File: src/packages/cli/src/Init.ts [hunk 3]
```diff
@@ -87,13 +87,31 @@ export class Init implements Command {
...
     
-    ${chalk.bold('Usage')}
+  ${chalk.bold('Usage')}
 
-      ${chalk.dim('$')} prisma init
+    ${chalk.dim('$')} prisma init [options]
+
```
Comment (janpio) [inline src/packages/cli/src/Init.ts:96]:
```suggestion

```
😝

**Comment by Jolg42** [comment]
This was shipped in 2.26.0 see how to use it in the release notes at https://github.com/prisma/prisma/releases/tag/2.26.0

**Pull request merged**